.github/workflows/sidecar-check.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2020-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Sidecar Digest validation
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  build:
+    name: Validate digests for sidecar images
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        name: "Checkout source code"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: "Set up QEMU"
+        uses: docker/setup-qemu-action@v1
+      -
+        name: "Set up Docker Buildx"
+        uses: docker/setup-buildx-action@v1
+      -
+        name: "Docker quay.io Login"
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - 
+        name: Login to docker.io
+        uses: docker/login-action@v1
+        with:
+          registry: docker.io
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Install skopeo
+        run: |
+          . /etc/os-release
+          echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
```

Comment:
wow, we're using opensuse + ubuntu here? shame we can't use a single OS like UBI8 :)

Reply:
well looks like there are no runners with ubuntu 20.10 yet, that have skopeo repos configured in them
so I guess it's either this, or running skopeo from some docker container (or using curl instead?)

Reply:
https://github.com/redhat-developer/codeready-workspaces/blob/crw-2-rhel-8/product/util.groovy#L353-L361 is another way to get skopeo from the official RH released container registry.redhat.io/rhel8/skopeo